doc_src/index.hdr.in

```diff
@@ -17,7 +17,199 @@
+
+Example:
+
+\fish
+# Every line of the `ls` command is immediatelly passed on to the `grep` command
+>ls -l | grep "my topic"
+\endfish
+
+- Commands can pass on all their output as a chunk</b>: the output stream of one command is bundled and taken as data argument for the second command. This is called command substitution, see <a href="#expand-command-substitution">Command Substitution</a>.
+
+Example:
+\fish
+# the output of the inner `ls` command is taken as the input argument for the outer `echo` command
+>echo (ls a*)
+\endfish
+
+\subsection shebang-line Shebang Line
+
+Since script for shell commands can be written in many different languages, they need to carry information about what interpreter is needed to execute them: For this they are expected to have a first line, the shebang line, which names an executable for this purpose:
+
+Example: 
+
+A scripts written in `bash` it would need a first line like this:
+
+```
+#!/bin/bash
+```
+
+- this line tells the shell to execute the file with the *bash* interpreter, that is located at the path `/bin/bash`.
+
+For a script, written in another language, just replace the interpreter `/bin/bash` with the language interpreter of that other language (for example `/bin/python` for a `python` script)
+
+This line is only needed when scripts are executed by another interpreter, so for *fish* internal commands, that are executed by *fish* the shebang line is not necessary.
+
+<a href="#shell"><-</a><hr>
+
+
+\section install Installation and Start
+
+This section is on how to install, uninstall, start and exit a *fish* shell and on how to make *fish* the default shell:
+
+- <a href="#installation">Install</a>: How to install *fish*
+- <a href="#default-shell">Default Shell</a>: How to switch to *fish* as the default shell
+- <a href="#start">Starting and Exiting</a>: How to start and exit a *fish* shell
+- <a href="#uninstall">Uninstall</a>: How to uninstall *fish*
+- <a href="#bash">Executing Bash</a>: How to execute *bash* commands in *fish*
+
+\subsection installation Installation
+
+Instructions for installing fish are on the <a href="https://fishshell.com/">fish homepage</a>. Search that page for "Go fish". 
+
+To install the development version of *fish* see the instructions at the <a href="https://github.com/fish-shell/fish-shell">project's GitHub page</a>.
+
+\subsection default-shell Default Shell
+
+You can make *fish* your default shell by adding *fish*'s  executable in two places:
+- add `/usr/local/bin/fish`  to  `/etc/shells`
+- change your default shell with `chsh -s` to `/usr/local/bin/fish`
+
+For for detailed instructions see <a href="tutorial.html#tut_switching_to_fish">Switching to *fish*</a>.
+
+\subsection uninstall Uninstalling
+
+For uninstalling *fish*: see <a href="faq.html#faq-uninstalling">FAQ: Uninstalling *fish*</a>
```

Comment:
If you wanted, you could move the FAQ entry on uninstalling here, and make the FAQ point here instead.

Reply:
Okay, I will do this, since it makes sense to me: all that is included in the docs does not be in the FAQs any more, since they may be seen as more of a temporary collection of what else could be asked about fish.